src/app/admin/invite/admin-users.actions.ts

```diff
@@ -12,6 +12,14 @@ export const adminInviteUserAction = adminAction(async (invite) => {
     const client = await clerkClient()
     let clerkUserId = ''
 
+    // Check if a user with this email already exists in Clerk
+    const existingUsers = await client.users.getUserList({
+        emailAddress: [invite.email],
+    })
+    if (existingUsers.data.length > 0) {
+        throw new SanitizedError(`A user with the email address '${invite.email}' already exists.`)
+    }
+
```

Comment:
I get what you're doing here, but doesn't clerk does that for us when we submit a dupe.  I just tested and saw:

<img width="473" alt="image" src="https://github.com/user-attachments/assets/4db3628d-3f8f-4c8a-bdf8-0bb211650f4e" />

I do think we could do a little better error handling than a toast message though, maybe check what I just did in #121 

https://github.com/safeinsights/management-app/pull/121/files#diff-eaa2f04c5b07b4ec606e1fe838bf0c6247c6733918fb0ce89b5cd521cbfa277fR32-R55

Reply:
I will try that and refactor, thanks! I'm also somehow mind-stuck by the weird testing errors here.